### PR TITLE
check IsNotExist error when validating mountpoint

### LINKED ISSUE #9952: rbd volume target mount point is created after it's checked for existence
I find [this code](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/volume/rbd/rbd_util.go#L103) questionable, as the check always fails when directory doesn't exist, and it's being created [a few lines later on](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/volume/rbd/rbd_util.go#L111).

### PR DESCRIPTION
**Pull request by rootfs**
This fixes #9952

### CODE REVIEW

File: pkg/volume/rbd/rbd_util.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func (util *RBDUtil) AttachDisk(rbd rbd) error {
 	// mount it
 	globalPDPath := rbd.manager.MakeGlobalPDName(rbd)
 	mountpoint, err := rbd.mounter.IsMountPoint(globalPDPath)
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```
Comment (pmorie) [inline pkg/volume/rbd/rbd_util.go]: I would add a note saying that the mountpoint not existing is a contingency.

Do you want this method to continue running if there's already a mountpoint for this volume?

Comment (rootfs) [inline pkg/volume/rbd/rbd_util.go]: yes, if directory is not there, that indicates it is the first time to create then mountpoint. the method should continue to run. the mountpoint is later created.

Comment (pmorie) [inline pkg/volume/rbd/rbd_util.go]: K, then I would add a comment there to that effect.

**Comment by saad-ali** [comment]
v1.0 triage needed: Marked the associated issue #9952 as v1.0-candidate

**Comment by satnam6502** [comment]
@k8s-bot ok to test

**Pull request merged**